.github/workflows/test_and_deploy.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      # fail-fast: true
```

Comment:
```suggestion
      fail-fast: true
```

Reply:
We need to wait for the CI checks to finish running, and see if they all successfully  create conda environments. Then we can commit this suggestion, and merge the PR.

Note: we are still expecting to see pytest failures here, which will be fixed by https://github.com/dask/dask-image/pull/366. But if pytest runs at all, then the conda environment definitely was created properly.